host/contracts/integrity_test.go

```diff
@@ -80,7 +80,7 @@ func TestCheckIntegrity(t *testing.T) {
 	defer node.Close()
 
 	am := alerts.NewManager()
-	s, err := storage.NewVolumeManager(node.Store(), am, node.ChainManager(), log.Named("storage"))
+	s, err := storage.NewVolumeManager(node.Store(), am, node.ChainManager(), log.Named("storage"), DefaultSettings.SectorCacheSize)
```

Comment:
I wasn't setting the value to 0 because of this this FAIL: TestCheckIntegrity (0.12s)
    integrity_test.go:85: failed to initialize cache: must provide a positive size
FAIL

Reply:
Yep, switch it back.